soco/config.py

```diff
@@ -31,6 +31,16 @@
 """
 
 
+EVENT_LISTENER_IP = None
+"""The IP on which the event listener listens.
+
+The default is None. If unset it will detect.
```

Comment:
Please change to "The default of None means that the relevant IP address will be detected automatically"